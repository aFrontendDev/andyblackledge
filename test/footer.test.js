import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import renderer from 'react-test-renderer';
import Footer from '../app/components/_footer/footer';


it('should render the footer', () => {
  const tree = renderer.create(
    <Footer  />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});