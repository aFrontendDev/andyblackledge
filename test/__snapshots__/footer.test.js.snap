// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the footer 1`] = `
<footer
  className="footer"
  role="footer"
>
  <small
    className="footer__copyright"
  >
    Â© Copyright Some Site 2017. All Rights Reserved.
  </small>
</footer>
`;
